src/Microsoft.DotNet.Watcher.Core/FileSystem/FileWatcher.cs

```diff
@@ -200,7 +190,7 @@ private static string ResolveRootDirectory(string projectPath)
 
             while (di.Parent != null)
             {
-                var globalJsonPath = Path.Combine(di.FullName, "global.json");
+                var globalJsonPath = Path.Combine(di.FullName, "project.json");
```

Comment:
Actually the whole function should be renamed to `ResolveProjectDirectory` since we are only looking for the root of the project instead of the root of the solution.